templates/home/api-en.html

```diff
@@ -0,0 +1,217 @@
+		&lt;/div&gt;
+	&lt;/div&gt;
+	</code></pre>
+	<h5>Provisional</h5>
+	<p>Support for a set of predefined color to match the season. The intend is to set the color of the inner box with the site title to match the promotional banner background color.</p>
+	<p>The promotional color theme are set on the <code>div.home-banner</code>. Here the supported theme with their corresponding CSS class name</p>
+	<ul>
+		<li>(default) Sky - <code>sky</code></li>
+		<li>Earth - <code>earth</code></li>
+		<li>Vegetation - <code>vegetation</code></li>
+		<li>Night - <code>night</code></li>
+		<li>Forest - <code>forest</code></li>
+		<li>Autumn - <code>autumn</code></li>
+	</ul>
+</details>
+
+<h3 id="your-government">Your government</h3>
+<dl>
+	<dt>Version</dt>
+	<dd>2.0</dd>
+	<dt>Updated</dt>
+	<dd>2022-04-27</dd>
+	<dt>Notes</dt>
+	<dd>Links description was removed.</dd>
+</dl>
+<p>There are 8 links in this section presented in 2 columns.</p>
+<pre><code>
+&lt;section class="home-your-gov gc-srvinfo well well-sm brdr-0"&gt;
+	&lt;div class="container"&gt;
+		&lt;h2&gt;Your government&lt;/h2&gt;
+		&lt;div class="row mrgn-bttm-lg"&gt;
+			&lt;div class="col-lg-7 col-xs-12 col-md-6"&gt;
+				&lt;ul class="row wb-eqht-grd list-unstyled small"&gt;
+					&lt;li class="col-xs-12 col-sm-6 mrgn-tp-sm mrgn-bttm-sm"&gt;
+						&lt;a href="https://www.canada.ca/en/contact.html"&gt;Contact us&lt;/a&gt;
+					&lt;/li&gt;
+					&lt;li class="col-xs-12 col-sm-6 mrgn-tp-sm mrgn-bttm-sm"&gt;
+						&lt;a href="https://www.canada.ca/en/news.html"&gt;News&lt;/a&gt;
+					&lt;/li&gt;
+					&lt;li class="col-xs-12 col-sm-6 mrgn-tp-sm mrgn-bttm-sm"&gt;
+						&lt;a href="https://pm.gc.ca/en"&gt;Prime Minister&lt;/a&gt;
+					&lt;/li&gt;
+					&lt;li class="col-xs-12 col-sm-6 mrgn-tp-sm mrgn-bttm-sm"&gt;
+						&lt;a href="https://www.canada.ca/en/government/dept.html"&gt;Departments and agencies&lt;/a&gt;
+					&lt;/li&gt;
+					&lt;li class="col-xs-12 col-sm-6 mrgn-tp-sm mrgn-bttm-sm"&gt;
+						&lt;a href="https://www.canada.ca/en/government/system.html"&gt;About government&lt;/a&gt;
+					&lt;/li&gt;
+					&lt;li class="col-xs-12 col-sm-6 mrgn-tp-sm mrgn-bttm-sm"&gt;
+						&lt;a href="https://open.canada.ca/en"&gt;Open government and data&lt;/a&gt;
+					&lt;/li&gt;
+					&lt;li class="col-xs-12 col-sm-6 mrgn-tp-sm mrgn-bttm-sm"&gt;
+						&lt;a href="https://www.canada.ca/en/government/publicservice.html"&gt;Working for the government&lt;/a&gt;
+					&lt;/li&gt;
+					&lt;li class="col-xs-12 col-sm-6 mrgn-tp-sm mrgn-bttm-sm"&gt;
+						&lt;a href="https://www.canada.ca/en/government/system/laws.html"&gt;Treaties, laws and regulations&lt;/a&gt;
+					&lt;/li&gt;
+				&lt;/ul&gt;
+			&lt;/div&gt;
+		&lt;/div&gt;
+	&lt;/div&gt;
+&lt;/section&gt;
+</code></pre>
+<h4>Previous version</h4>
+<details>
+	<summary>Deprecated - Version 1.0</summary>
+	<p>There were four links with descriptions.</p>
+	<pre><code>
+	&lt;section class="home-your-gov gc-srvinfo well well-sm brdr-0"&gt;
```

Comment:
Is this still qualify to be a "gc-srvinfo" component. Considering there is no description anymore I don't think that piece of code qualify to be a gc service and information component.

Reply:
Changed.Thanks!